### ISSUE TITLE
[jshow] Where is the case study tab for? Can we disable it if we have nothing in it?

### ISSUE THREAD

**Issue by zero-24** [joomla/joomla-websites#547]
Is there a way to disable that tab if there is nothing in it?

![image](https://cloud.githubusercontent.com/assets/2596554/18132391/f0b892d0-6f96-11e6-8935-1f0a21fb0afc.png)

**Comment by brianteeman** [comment]
My understanding is that is where you write about how the site was built and what you used. As opposed to the description tab which is for who the site is for etc.

I know I plan to add case study sections to my sites on the showcase but haven't had the time yet

**Comment by zero-24** [comment]
This was not against you personally @brianteeman this was just the example i snipped with the snipping tool we have the same for the others i have looked in.

**Comment by brianteeman** [comment]
I didn't take it personally

**Comment by jeckodevelopment** [comment]
I noticed that too.. most of people won't include the Case Study, so it could be useful to see the tab only when needed.

**Comment by AlisonAMG** [comment]
"My understanding is that is where you write about how the site was built and what you used. As opposed to the description tab which is for who the site is for etc."

That would be a helpful notation to place there for when we are filling in our details. I didn't think of it that way but it's a great idea.

**Comment by sebastienlapoux** [comment]
Thank you!
We added some help text. Please let us know if it's correct or if you have better english sentence.

Good point Alison! Because this field is not require, we will display it only if a value.

**Comment by zero-24** [comment]
Can you double check that it got displayed only if ot has a value?

**Comment by sebastienlapoux** [comment]
@zero-24 sorry i don't understand your answer.
I wrote just before "we will display it only if a value."

**Comment by zero-24** [comment]
![wp_ss_20160831_0001](https://cloud.githubusercontent.com/assets/2596554/18144703/380e6c6e-6fc8-11e6-875b-058ec42f604b.png)
Hmm it looks like it is not one the live site? Or do i miss something?

**Comment by mbabker** [comment]
He's saying they haven't done it yet but will.

**Comment by brianteeman** [comment]
"We will" ===
"We have not done it yet but will do it"

**Comment by zero-24** [comment]
Ahh ok. Sorry that i missunderstand you.

**Comment by brianteeman** [comment]
Germans, French and Americans all talking in a foreign language....

**Comment by jeckodevelopment** [comment]
@brianteeman let me share with you a video
https://youtu.be/93PR3HmtzjA?list=PLSwFVRVZ3joTxUoEcln2XEmU7nL5MDgUP

**Comment by sebastienlapoux** [comment]
Hi,
Case study tab displayed only if a value.
* thanks to delete your navigator cache before to test

**Comment by MATsxm** [comment]
OK here -Thanks
Closing - Reopen if needed
